---
description: LOCALDB_ERROR_UNKNOWN_INSTANCE
title: LOCALDB_ERROR_UNKNOWN_INSTANCE | Документация Майкрософт
ms.custom: ''
ms.date: 03/03/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 67bca4f1-915d-46d4-9293-3742c16930a1
author: stevestein
ms.author: sstein
ms.openlocfilehash: 891c38344e53580c6bba44924d018ccf1bce9f02
ms.sourcegitcommit: e700497f962e4c2274df16d9e651059b42ff1a10
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 08/17/2020
ms.locfileid: "88470543"
---
# <a name="localdb_error_unknown_instance"></a>LOCALDB_ERROR_UNKNOWN_INSTANCE
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>Сведения  
  
| attribute | Значение |
| --------- | ----- |
|Название продукта|SQL Server|  
|Идентификатор события|263|  
|Источник события|Среда выполнения локальной базы данных SQL Server 12.0|  
|Компонент|API среды выполнения локальной базы данных|  
|Текст сообщения|Указанный экземпляр локальной базы данных не существует.|  
  
## <a name="explanation"></a>Объяснение  
 Указанный экземпляр не существует.  
  
## <a name="user-action"></a>Действие пользователя  
 Проверьте правильность указанного имени экземпляра.  
  
  
